care/facility/api/serializers/patient_sample.py

```diff
@@ -73,11 +73,14 @@ def update(self, instance, validated_data):
         if choice == "COMPLETED" and not validated_data.get("result"):
             raise ValidationError({"result": [f"is required as the test is complete"]})
 
-        if validated_data.get("status") == PatientSample.SAMPLE_TEST_FLOW_MAP["SENT_TO_COLLECTON_CENTRE"]:
+        if (
+            validated_data.get("status") == PatientSample.SAMPLE_TEST_FLOW_MAP["SENT_TO_COLLECTON_CENTRE"]
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Wrong hanging indentation before block (add 4 spaces).](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5344889)

Reply:
This conflicts with the autoformatting applied by Black...